### PR TITLE
Fix version number in docs and prepare new release

### PR DESCRIPTION
**Pull request by llucax**
- Clear release notes
- Fix outdated version in migrating and upgrading docs
- Clear release notes for release
- Bump version number in the docs

### CODE REVIEW

**Comment by Marenz** [comment]
CI failing

**Comment by llucax** [comment]
Yeah, missed updating the tests. Should pass now. :crossed_fingers:

**Comment by llucax** [comment]
Green

**Review by Marenz — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,85 +1,5 @@
...
-  * Fix copying of `Config` and `CommandOptions` objects.
-
-### Cookiecutter template
-
-- Now the CI workflow will checkout the submodules.
-
-- Fix adding of an empty keyword.
-
-- Don't distribute development files in the source distribution.
+- Fix outdated version number in the migrating and upgrading documentation.
```
Comment (daniel-zullo-frequenz) [inline RELEASE_NOTES.md:5]:
```suggestion
- Fix outdated version number in migrating and upgrading documentation.
```

**Review by daniel-zullo-frequenz — COMMENTED** [review]
I only have an optional cosmetic comment and LGTM

**Comment by llucax** [comment]
> I only have an optional cosmetic comment and LGTM

I would have liked to accept those changes. If auto-merge is enabled and you make some comments, please either disable it or use the "request changes" type of comments to auto-merge is not triggered.

**Comment by llucax** [comment]
To avoid another round for such a small change I will just fix it in the GitHub release description instead.

**Pull request merged**